.circleci/config.yml

```diff
@@ -368,7 +368,7 @@ jobs:
           name: Check packages
           command: expotools check-packages
 
-  web_test_suite:
+  web-Test-Suite:
```

Comment:
Since these are programmatic IDs I would keep them all lowercase and formatted the same as the other job names